css-module-bikeshed/Overview.bs

```diff
@@ -159,3 +159,7 @@ Internal display model: the 'foo' property {#the-foo-property}
 			DOMString baz = "qux";
 		};
 	</pre>
+
+<h2 class=no-num id=priv-sec>Privacy and Security Considerations</h2>
+
+Make some considerations about privacy and security.
```

Comment:
That it can be easily forgotten is one reason I opened issue #240.
Obviously very few editors actually added this section to their specs so far. So it seemed to me the Bikeshed warning is not enough and I was wondering about how to push people more to add that section.

It's ok for me to add an issue instead, though before I do that I'm asking about this at https://github.com/w3c/csswg-drafts/issues/240#issuecomment-228660782.

Sebastian